### PR TITLE
CIV-5851 payment date

### PR DESCRIPTION
**Pull request by laperla11**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-5851

### Change description ###
This screen will allow to LR to select what type of repayment plan they want.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: codecept.conf.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ exports.config = {
     './e2e/tests/*_test.js',
```
Comment (laperla11) [inline codecept.conf.js:4]: changes from Kenneth's story

File: e2e/api/steps_LRspecCui.js [hunk 2]
```diff
@@ -0,0 +1,352 @@
+const config = require('../config.js');
```
Comment (laperla11) [inline e2e/api/steps_LRspecCui.js:1]: changes from Kenneth's story

File: e2e/fixtures/events/defendantResponseSpecCui.js [hunk 3]
```diff
@@ -0,0 +1,516 @@
+const {listElement} = require('../../api/dataHelper');
```
Comment (laperla11) [inline e2e/fixtures/events/defendantResponseSpecCui.js:1]: changes from Kenneth's story

File: e2e/run-spec-functional-tests.sh [hunk 4]
```diff
@@ -5,4 +5,7 @@ echo "Run Api Functional tests for api-spec"
 yarn test:api-spec
```
Comment (laperla11) [inline e2e/run-spec-functional-tests.sh:5]: changes from Kenneth's story

File: e2e/run-api-ff-enabled-functional-tests.sh [hunk 5]
```diff
@@ -11,3 +11,6 @@ yarn test:api-spec
```
Comment (laperla11) [inline e2e/run-api-ff-enabled-functional-tests.sh:11]: changes from Kenneth's story

File: e2e/tests/api_tests/lrspec_cui/api_1v1_spec_cui_test.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
```
Comment (laperla11) [inline e2e/tests/api_tests/lrspec_cui/api_1v1_spec_cui_test.js:1]: changes from Kenneth's story

File: package.json [hunk 7]
```diff
@@ -28,6 +28,7 @@
     "test:api-spec-1v2": "MOCHAWESOME_REPORTFILENAME=api-spec npx codeceptjs run --grep @api-spec-1v2 --reporter mocha-multi --verbose",
     "test:api-spec-1v1": "MOCHAWESOME_REPORTFILENAME=api-spec npx codeceptjs run --grep @api-spec-1v1 --reporter mocha-multi --verbose",
     "test:api-spec-small": "MOCHAWESOME_REPORTFILENAME=api-spec npx codeceptjs run --grep @api-spec-small --reporter mocha-multi --verbose",
+    "test:api-spec-cui": "MOCHAWESOME_REPORTFILENAME=api-spec-cui npx codeceptjs run --grep @api-spec-cui --reporter mocha-multi --verbose",
```
Comment (laperla11) [inline package.json:31]: changes from Kenneth's story

File: e2e/fixtures/events/claimantResponseSpecCui.js [hunk 8]
```diff
@@ -0,0 +1,128 @@
+const {listElement} = require('../../api/dataHelper');
```
Comment (laperla11) [inline e2e/fixtures/events/claimantResponseSpecCui.js:1]: changes from Kenneth's story

File: ccd-definition/CaseEventToComplexTypes/ClaimantResponseLRspec-CUI-nonprod.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+[
+
```
Comment (lfazelyniaHMCTS) [inline ccd-definition/CaseEventToComplexTypes/ClaimantResponseLRspec-CUI-nonprod.json:2]: We can probably delete this file as we don't use it
Kenneth will be adding it in anyway

File: ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json [hunk 10]
```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CLAIMANT_RESPONSE_SPEC",
+    "CaseFieldID": "applicant1RepaymentOptionForDefendantSpec",
```
Comment (lfazelyniaHMCTS) [inline ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json:5]: Please don't forget to remove/overwrite this with my version once my changes get merged

Comment (laperla11) [reply ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json:5]: definitely will do it

**Review by lfazelyniaHMCTS — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

File: ccd-definition/CaseEventToComplexTypes/ClaimantResponseLRspec-CUI-nonprod.json [hunk 11]
```diff
@@ -0,0 +1,3 @@
...
+]
```
[the remark this answers is not in the document]
Comment (laperla11) [reply ccd-definition/CaseEventToComplexTypes/ClaimantResponseLRspec-CUI-nonprod.json:3]: maybe we need to use it in future

**Pull request closed without merging**